openassessment/xblock/static/js/src/lms/oa_course_items_listing.js

```diff
@@ -52,12 +52,39 @@ export class CourseItemsListingView {
       url: 'url_grade_available_responses',
     });
 
+    const context = this.data.CONTEXT || {};
+
+    const esgEnabled = context.ENHANCED_STAFF_GRADER;
+    const esgUrl = context.ORA_GRADING_MICROFRONTEND_URL;
+
+    const UnitNameCell = Backgrid.UriCell.extend({
```

Comment:
I would look at making this sort of change around line 30, where we set up the "click number to go to page" sort of thing. Of course since this is an external page we'll want to just make the link an actual link rather than what they're currently doing there.

Reply:
The variable name `AssessmentCell` was what drove me into creating a new one. Additionally, it had a certain level on string comparing.